src/components/NotesMindMap/NotesMindMapReducer.js

```diff
@@ -164,6 +165,12 @@ export const notesMindMapReducer = (state = defaultState, { type, data }) => {
       return handleChangeParentNoteRequestSuccess();
     case CHANGE_PARENT_REQUEST_FAIL_ACTION:
       return { ...state, isChangeParentModeActive: false };
+    case SEARCH_RESULT_CLICKED:
+      return {
+        nodes: [data.note],
+        edges: data.edges,
```

Comment:
let's just set the edges to `[]` here, instead of passing `[]` from action.

Action is `SEARCH_RESULT_CLICKED` and it passes the clicked search result as payload. Reducer decides what to do with edges. The action just says what happened: which search result was clicked.